core/sim/rtl_sim/run/run_dma_periph

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
not sure we need this file? Id like to avoid bloating the code base w even more files.. So unless it's strictly needed, let's remove this file?

Reply:
I created a separate file because this test (unlike the others) needs to be run with `__SANCUS_SIM=1`, and this seemed like a cleaner approach. But if you want, I can embed it in the `run_all_sancus` file, and just activate this flag before this test.

Reply:
yes better to keep it in run_all_sancus and set the flag for this specific test case I think ^^